src/bindActionCreators.ts

```diff
@@ -73,6 +108,22 @@ export default function bindActionCreators(
   }
 
   const boundActionCreators: ActionCreatorsMapObject = {}
+
+  /**
+     * #SCC
+     *
+     * I've always learned to avoid using for...in for iterating over object keys because
+     * it also iterates over all properties in that object's prototype chain. In this case there
+     * is an if statement that checks we are actually getting an action creator like we expect, but
+     * the safer way that I've always done this is to use Object.keys to get an array of all
+     * keys on the object and then just use a map or forEach or something. The for...in syntax
+     * is nice which is my guess as to why they're using it here, but I think there is something
+     * to be said about how nice it is to use array functions that enforce object immutability like
+     * map and reduce. It's also kind of interesting to see object mutation in the source code of a library
+     * that enforces state immutability. Maybe we don't need to be so dogmatic about it?
+     *
+     * - Marko Skugor
+    */
```

Comment:
I can think of 2 reasons that the authors' went with the `for...in` :

1. Performance  - loops are more performant than higher-order array methods 
2. Compatibility - `object.keys`[ isn't support universally](https://caniuse.com/?search=object.keys)

My gut says it isn't about performance and is more likely a compatibility thing, kind of like the use of string concatenation instead of  template literals in other places.